python_ta.egg-info/requires.txt

```diff
@@ -0,0 +1,34 @@
+astroid~=3.1.0
```

Comment:
Remove this file (it should not be committed, and probably you can delete it from your local repository as well)